packages/plugin-auto-tracking-browser/src/auto-tracking-plugin.ts

```diff
@@ -140,15 +196,15 @@ export const autoTrackingPlugin = (options: Options = {}): BrowserEnrichmentPlug
         });
       }
     };
-    const allElements = Array.from(document.body.querySelectorAll(tagAllowlist.join(',')));
+    const allElements = Array.from(document.body.querySelectorAll(DEFAULT_TAG_ALLOWLIST.join(',')));
```

Comment:
cssSelectorAllowList instead?

Reply:
Selector is too dynamic, e.g., `[":scope>span", "button.abc"]`, I think a simple `join` combining might cause unexpected results. We can technically add another loop, iterating through all selectors then check, but there might have duplicates depending on the quality of selectors. I filter with the default tags at first then filter with any custom configurations later on. Do you have any concern on this?